docs/template_plugin/tests/functional/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ find_package(OpenCV QUIET COMPONENTS core imgproc)
 if(OpenCV_FOUND)
     message("-- Reference preprocessing: OpenCV tests are enabled")
     target_compile_definitions(${TARGET_NAME} PRIVATE OPENCV_TEMPLATE_TESTS)
-    target_link_libraries(${TARGET_NAME} PRIVATE opencv_imgproc opencv_core)
+    target_link_libraries(${TARGET_NAME} PRIVATE opencv_imgproc opencv_core ov_core_dev_api)
```

Comment:
ov_core_dev_api should better not be in the condition of "OpenCV_FOUND".

Reply:
resolved